pillar/edx/xqwatcher.sls

```diff
@@ -98,6 +98,44 @@ edx:
       {% endfor %}
       {% endif %}
       {% endfor %}
+      {% set 686_xqueue_creds = salt.vault.read('secret-residential/global/course-686-xqueue-credentials') %}
```

Comment:
Those credentials are not in vault yet, correct?

Reply:
They are

Reply:
Looks like they're under `secret-residential/global/course-686x-grader-xqueue-credentials`